src/style.css

```diff
@@ -0,0 +1,45 @@
+.container {
+  margin: 0 ;
+  padding: 0 ;
+  box-sizing: border-box ;
+}
+
+.bod {
+  margin: 0 ;
+  padding: 0 ;
+}
+
+.text {
+  display: none ;
+}
+
+.stars__star {
+  width: 16px ;
+  height: 16px ;
+  padding-right: 4px ;
+  background: url("images/star.svg") 2px 2px /13px no-repeat ;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex ;
+  align-items: center ;
+}
+
+.stars--1 :first-child,
+.stars--2 :first-child,
+.stars--2 :nth-child(2),
+.stars--3 :first-child,
+.stars--3 :nth-child(2),
+.stars--3 :nth-child(3),
+.stars--4 :first-child,
+.stars--4 :nth-child(2),
+.stars--4 :nth-child(3),
+.stars--4 :nth-child(4),
+.stars--5 .stars__star {
+  background: url("images/star-active.svg") 2px 2px / 13px no-repeat ;
+}
```

Comment:
use [:nth-child](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child) with `n` to set the correct background for each modificator instead of setting it for every single star